ext/jtwig/src/main/java/org/mvcspec/ozark/ext/jtwig/JtwigViewEngine.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mvcspec.ozark.ext.jtwig;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.mvc.engine.ViewEngineContext;
+import javax.mvc.engine.ViewEngineException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jtwig.web.servlet.JtwigRenderer;
+import org.mvcspec.ozark.engine.ViewEngineBase;
+
+/**
+ * @author Daniel Dias
+ */
+@ApplicationScoped
+public class JtwigViewEngine extends ViewEngineBase {
+
+    private JtwigRenderer jtwigRenderer;
+
+    @PostConstruct
+    public void init() {
+        jtwigRenderer = JtwigRenderer.defaultRenderer();
+    }
+
+    public boolean supports(String view) {
+        return view.endsWith(".twig.html") || view.endsWith(".twig");
+	}
+
+    public void processView(ViewEngineContext context) throws ViewEngineException {
+        try {
+    	    Map<String, Object> model = new HashMap<>(context.getModels());
+            model.put("request", context.getRequest(HttpServletRequest.class));
```

Comment:
Could you check the indentation of all files. A few files (also the `pom.xml`) and not indented correctly.